example/android/app/src/main/AndroidManifest.xml

```diff
@@ -10,7 +10,8 @@
     android:icon="@mipmap/ic_launcher"
     android:label="@string/app_name"
     android:theme="@style/AppTheme"
-    tools:ignore="GoogleAppIndexingWarning">
+    tools:ignore="GoogleAppIndexingWarning"
+    android:networkSecurityConfig="@xml/network_security_config">
```

Comment:
Can we please add this:
`// This is for Detox test framework`

So we don't forget why this is needed

Reply:
Done